src/app/chat/page.tsx

```
@@ -64,6 +64,7 @@ import { useNotifications } from "@/hooks/useNotifications";
 import { apiCache, CACHE_TTL } from "@/lib/cache-utils";
 import { listInboxWithFilters } from "@/lib/inbox-client";
 import { useChatSurfaceController } from "./hooks/useChatSurfaceController";
+import { logger } from "@/lib/client-logger";
```

ESLint

[error] 'logger' is defined but never used.

(unused-imports/no-unused-imports)